src/main/groovy/quartz/QuartzGrailsPlugin.groovy

```diff
@@ -32,7 +32,7 @@ import org.springframework.scheduling.quartz.SchedulerFactoryBean
 class QuartzGrailsPlugin extends Plugin {
 
     // the version or versions of Grails the plugin is designed for
-    def grailsVersion = "6.0.0 > *"
+    def grailsVersion = "7.0.0-SNAPSHOT > *"
 
 	def watchedResources = "file:./grails-app/jobs/**/*Job.groovy"
```

Comment:
Down below where authorEmail is set, we've been unsetting them as we upgrade these plugins.

Reply:
I removed author and authorEmail, I wasn't sure about so of the other items like Title and description.  If you would like I can remove them

Reply:
We typically leave the author, just not their email.  I'd leave everything as is.